doc/whats_new.rst

```diff
@@ -68,6 +68,7 @@ Authors
 People who contributed to this release (in alphabetical order):
 
 * Alexandre Gramfort
+* Sophie Herbst
```

Comment:
you don't have to add your name here. It will be done when we make the next release